src/sources/prometheus/parser.rs

```diff
@@ -78,6 +84,10 @@ fn reparse_groups(
                 for (key, metric) in metrics {
                     let tags = combine_tags(key.labels, tag_overrides.clone());
 
+                    let shard_name = tags.get("shardName").unwrap_or("unknown").to_string();
```

Comment:
Is it possible to make this `shardName` configurable at the source level?

Reply:
Wondering you mean configurable from source level is from this metric configuration? This metric should already have "xxx--general_1" shardName from scraping. This is a label for input source shardName we want to understand the load volume from each input shard. such nephos_1 is sending x series/seconds.

Reply:
I mean, having a new option in https://vector.dev/docs/reference/configuration/sources/prometheus_scrape/ since `shardName` is very databricks specific. It could be called `latency_label` and set to `"shardName"`